policies/common/Pod.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kyverno.io/v1
```

Comment:
This is the only thing I really intended to add in this PR.

Reply:
This _should_ work - it will however set the env vars to `""` when the helm vars are empty which could overwrite them in some cases? Uncertain if that's an issue.

Reply:
I've improved it so it only applies the env vars when they aren't empty. I'll confirm that it works tomorrow.